Wikipedia.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I think you can remove this now that we've gotten rid of the outer workspace file.

Reply:
I tried to update this branch to `main`, but think I made things worse. Going to close this out, and open a new PR. Anyhow, this comment should be addressed by recreating this off `main`.